openedx/core/djangoapps/catalog/management/commands/tests/test_cache_programs.py

```diff
@@ -61,6 +74,7 @@ def detail_callback(request, uri, headers):
             content_type='application/json'
         )
 
+    @waffle.testutils.override_switch("populate-multitenant-programs", True)
```

Comment:
Minor: please stay consistent with the code that was already here. Single quotes, please, here and below.